src/GafferImageUI/ImageView.cpp

```diff
@@ -210,6 +230,8 @@ ImageView::ImageView( const std::string &name )
 	clampNode->minClampToPlug()->setValue( Color4f( 1.0f, 1.0f, 1.0f, 0.0f ) );
 	clampNode->maxClampToPlug()->setValue( Color4f( 0.0f, 0.0f, 0.0f, 1.0f ) );
 
+    addChild( new FloatPlug( "zoomLevel", Plug::In, 1.0f, 0.001, 100, Plug::Default & ~Plug::AcceptsInputs ) );
+
```

Comment:
To query the min/max of a plug, you can use `plug->minValue()` and `plug->maxValue()`. Is that what you meant?